NewFiles/simple.js

```diff
@@ -1 +1,9 @@
-//would someone like to make the equivalent javascript version of the Java class in JuanBecerra.java ?
\ No newline at end of file
+//would someone like to make the equivalent javascript version of the Java class in JuanBecerra.java ?
+
+//Does the below javascript function work for you?
```

Comment:
It does. 

If we were to make it more like the Java version, we could make a class (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)